src/utils/calculateTotalNumberOfPrizes.ts

```diff
@@ -1,11 +1,11 @@
-import { calculateNumberOfPrizesForIndex, PrizeTier } from '@pooltogether/v4-js-client'
+import { calculate, PrizeTier } from '@pooltogether/v4-client-js'
 
 export const calculateTotalNumberOfPrizes = (prizeTier: PrizeTier): number => {
-  return prizeTier.tiers.reduce((acc, curr, index) => {
-    if (curr) {
-      return acc + calculateNumberOfPrizesForIndex(prizeTier.bitRangeSize, index)
-    } else {
-      return acc
-    }
+  return prizeTier.tiers.reduce((totalNumberPrizes: number, currentTier: number, index: number) => {
```

Comment:
Nice to see words here, I've always disliked abbreviations in code (eg. `acc` and `curr`)